### PR TITLE
Give readers the permission to add notes

### PR DESCRIPTION
**Pull request by StefanFl**
solves #5581

There is no harm in letting users with a `Reader` role adding notes. Includes a missing permission check, which doesn't change a lot because a user need a `Reader` role anyway to call this view method. Nevertheless it shall be there for completeness.

### CODE REVIEW

**Comment by valentijnscholten** [comment]
Small update to the docs needed?

**Comment by StefanFl** [comment]
> Small update to the docs needed?

True, done. And thanks for your changes.

File: docs/content/en/usage/permissions.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Users can be assigned as members to Products and Product Types, giving them one
 | View Components             | x      | x      | x          | x     | x            |
 |                             |        |        |            |       |              |
 | View Note History           | x      | x      | x          | x     |              |
-| Add Note                    |        | x      | x          | x     |              |
+| Add Note                    | x      | x      | x          | x     |              |
 | Edit Note                   |        | x      | x          | x     |              |
 | Delete Note                 |        | (x) <sup>2)</sub> | x          | x     |              |
```
Comment (valentijnscholten) [inline docs/content/en/usage/permissions.md:73]:
```suggestion
| Edit Note                   | (x) <sup>2)</sub>       | x      | x          | x     |              |
| Delete Note                 | (x) <sup>2)</sub>       | (x) <sup>2)</sub> | x          | x     |              |
```

**Review by valentijnscholten — CHANGES_REQUESTED** [review]
the suggestion might be more accurate?

File: docs/content/en/usage/permissions.md (hunk 1 above)
Comment (StefanFl) [reply docs/content/en/usage/permissions.md]: Well spotted, done

**Review by valentijnscholten — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Pull request merged**